.travis.yml

```diff
@@ -82,6 +82,11 @@ jobs:
 
         -
             <<: *STANDARD_TEST_JOB
+            php: 7.1
+            env: PHP_CS_FIXER_TEST_USE_LEGACY_TOKENIZER=1 SYMFONY_VERSION="^4.0@dev"
```

Comment:
Legacy tokenizer has to be used in at least one job, but no need to have it multiple times - we want to test it, but it makes job slower.
Also, there is no need to have new job to handle Symfony4, we could reuse existing job.
Please combine both 7.1 jobs into one (so, basically, drop previous job)